File: LaGranjaSA/src/main/java/com/example/LaGranjaSA/modelo/Cliente.java
package com.example.LaGranjaSA.modelo;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Table(name = "cliente")
public class Cliente {
    @Id
    private String cedula;
    private String nombres;
    private String apellidos;
    private String direccion;
    private String telefono;

}

File: LaGranjaSAFront/LaGranjaFront/src/app/model/Cliente.ts
export class Cliente {
    public cedula: String;
    public nombres: String;
    public apellidos: String;
    public direccion: String;
    public telefono: String;
    constructor(cedula: String, nombres: String, apellidos: String, direccion: String, telefono: String ) {
        this.cedula = cedula;
        this.nombres = nombres;
        this.apellidos = apellidos;
        this.direccion = direccion;
        this.telefono = telefono;
    }
    
}